plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/model/DefaultGModelState.java

```diff
@@ -22,15 +22,21 @@
 import org.eclipse.emf.common.command.CommandStack;
 import org.eclipse.glsp.graph.GModelIndex;
 import org.eclipse.glsp.graph.GModelRoot;
+import org.eclipse.glsp.server.di.ClientId;
 import org.eclipse.glsp.server.internal.gmodel.commandstack.GModelCommandStack;
 
-public class GModelStateImpl implements GModelState {
+import com.google.inject.Inject;
 
-   private Map<String, String> options;
-   private String clientId;
-   private GModelRoot currentModel;
-   private BasicCommandStack commandStack;
-   private String editMode;
+public class DefaultGModelState implements GModelState {
+
+   @Inject()
```

Comment:
```suggestion
```
I think this can be omitted.

Reply:
:-1:

Reply:
Hm, you are right, sorry! I tested the changes locally and the editor still works fine. However, now I set a breakpoint and the clientId is indeed null. Interesting that it still works though :)
Thanks for reverting this change!